src/apps/workspaces/service/workspace.py

```diff
@@ -272,15 +271,7 @@ async def exists_by_owner_id(self, owner_id: uuid.UUID):
         if not exists:
             raise WorkspaceDoesNotExistError()
 
-    async def get_applets_roles_by_priority(self, owner_id, appletIDs):
-        # parse applet ids
-        try:
-            applet_ids = list(
-                map(uuid.UUID, filter(None, appletIDs.split(",")))
-            )
-        except ValueError:
-            raise InvalidAppletIDFilter
-
+    async def get_applets_roles_by_priority(self, owner_id, applet_ids):
```

Comment:
applet_ids is a list now?

Reply:
Forgot to add annotations. Interesting why mypy didn't catch it.
applet_ids are always list[uuid.UUID]. We had some strange case for parsing appletIDs in API call `managers_priority_role_retrieve`, which receives appletIDs like string. appletIDs should be converted to the list of uuids in Dependency level, but not in the service level. Ideally it should work like [here](https://github.com/ChildMindInstitute/mindlogger-backend-refactor/blob/develop/src/apps/workspaces/api.py#L122) , but it can possible break current logic on frontend.